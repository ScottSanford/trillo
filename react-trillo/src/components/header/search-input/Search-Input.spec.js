import React from 'react'
import { shallow } from 'enzyme'
import SearchInput from './Search-Input'

describe('SearchInput Component', () => {

	it('should match snapshot', () => {
		const wrapper = shallow(<SearchInput />)
		expect(wrapper).toMatchSnapshot()
	})
})