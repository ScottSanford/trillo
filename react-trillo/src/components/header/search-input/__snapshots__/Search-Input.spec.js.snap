// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput Component should match snapshot 1`] = `
<form
  action="#"
  className="search"
>
  <input
    className="search__input"
    placeholder="Search hotels"
    type="text"
  />
  <button
    className="search__button"
  >
    <svg
      className="search__icon"
    >
      <use
        xlinkHref="img/sprite.svg#icon-magnifying-glass"
      />
    </svg>
  </button>
  <div
    className="search__latest-searches"
  >
    <div
      className="search__title"
    >
      <span>
        Recent Searches
      </span>
      <button
        className="btn-inline"
      >
        View all
      </button>
    </div>
    <ul
      className="search__list"
    >
      <li
        className="search__query"
      >
        <svg
          className="search__icon"
        >
          <use
            xlinkHref="img/sprite2.svg#icon-back-in-time"
          />
        </svg>
        <span>
          Hotels near Albufeira
        </span>
      </li>
      <li
        className="search__query u--margin-bottom-small"
      >
        <svg
          className="search__icon"
        >
          <use
            xlinkHref="img/sprite2.svg#icon-back-in-time"
          />
        </svg>
        <span>
          Hotels in Barcelona
        </span>
      </li>
    </ul>
    <div
      className="search__title"
    >
      <span>
        What's Trending
      </span>
    </div>
    <ul
      className="search__list"
    >
      <li
        className="search__query"
      >
        <svg
          className="search__icon"
        >
          <use
            xlinkHref="img/sprite2.svg#icon-trending-up"
          />
        </svg>
        <span>
          Hotels near the Roman Colosseum
        </span>
      </li>
      <li
        className="search__query"
      >
        <svg
          className="search__icon"
        >
          <use
            xlinkHref="img/sprite2.svg#icon-trending-up"
          />
        </svg>
        <span>
          Cheap hotels in Europe
        </span>
      </li>
    </ul>
  </div>
</form>
`;
